pyscript.core/tests/integration/test_01_basic.py

```diff
@@ -301,7 +310,7 @@ def test_js_version(self):
         )
 
     # TODO: ... and we shouldn't: it's a module and we better don't leak in global
-    @pytest.mark.skip("DIFFERENT BEHAVIOUR: we don't expose pyscript on window")
+    @pytest.mark.skip("NEXT: we don't expose pyscript on window")
```

Comment:
`import pyscript` works in either *main* (window?) or *worker* so I am not sure I understand this skip message.

Reply:
if this is about leaking globally *pyscript* on window, we dont' do that anymore, and likely we shouldn't ... as we land a module, and we discussed already, and filed issues and MRs, aroud *not* leaking on global.